testdata/terraform/terraformproject/aws/asg/files/mongodb_bootstrap.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -xe
```

Comment:
Do we must have this file content?
Why don't we just put a short test in it? The same goes for the content fo the other terraform test files.
If possible, let's also rename this file to "f".